valhalla/userrequests/templates/userrequests/userrequest_list.html

```diff
@@ -139,6 +142,7 @@
   {{ block.super }}
   <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.15.1/moment.min.js"></script>
   <script src='https://cdnjs.cloudflare.com/ajax/libs/bootstrap-datetimepicker/4.17.43/js/bootstrap-datetimepicker.min.js'></script>
+  <script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.6.2/jquery.min.js"></script>
```

Comment:
You don't need to include jquery. It is already included in base.html (which this template inherits from)